docs/docs/60-new-docs/50-user-guide/20-how-to-guides/05-installing-kargo-cli.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_label: Installing Kargo CLI
+---
+
+# Installing Kargo CLI
+
+The Kargo CLI provides a command-line interface to manage Kargo efficiently, serving as
+a faster, automation-friendly alternative to the UI.
```

Comment:
Two things here:

> The Kargo CLI provides a command-line interface

This seems like a redundant statement, so I think we should shorten this.

> automation-friendly alternative

This is less obvious, but I think we are best avoiding statements like this because of [this](https://docs.kargo.io/new-docs/faqs#how-do-i-integrate-kargo-into-my-ci-pipelines).

```suggestion
The `kargo` CLI is an alternative to Kargo's web-based UI for managing Kargo projects.
```